### PR TITLE
VEN-906 | Show name on application list, adjust application lists

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description :sparkles:

* Add applicant name to berth and winter storage application list tables
* Shorten berth application list column texts
  * Hakemustyyppi -> Tyyppi
  * Uusi hakemus -> Uusi
  * Vaihtohakemus -> Vaihto
  * Vene- / talvisäilytyspaikka -> Venepaikka
* Unify code related to application lists

### Closes :no_good_woman:

* [VEN-906](https://helsinkisolutionoffice.atlassian.net/browse/VEN-906): FE - Add applicant name to the application list tables

### Related :handshake:

* [VEN-920](https://helsinkisolutionoffice.atlassian.net/browse/VEN-920): Fit admin UI to a minimum screen width

### Automated tests :gear:️

* Updated snapshots and mock data

### Manual testing :construction_worker_man:

* Applicant name should be a column on berth application list
* Applicant name should be a column on winter storage application list
* Berth application list table should be a bit narrower
* The listings should have no regressions (broken features)

### CODE REVIEW

File: src/features/winterStorageApplicationList/utils.ts [hunk 1]
```diff
@@ -1,62 +1,88 @@
...
+        boatWidth,
+        createdAt,
+        customer,
+        firstName,
+        id,
+        lastName,
+        municipality,
+        status,
+        winterStorageAreaChoices,
+      } = (edge as WINTER_STORAGE_APPLICATIONS_EDGE).node as WINTER_STORAGE_APPLICATIONS_NODE;
```
Comment (Nizar-Rahme) [inline src/features/winterStorageApplicationList/utils.ts:54]:
```suggestion
      if (!edge?.node) return acc;
      
      const {
        boatLength,
        boatModel,
        boatName,
        boatRegistrationNumber,
        boatType,
        boatWidth,
        createdAt,
        customer,
        firstName,
        id,
        lastName,
        municipality,
        status,
        winterStorageAreaChoices,
      } = edge.node;
```

**Review by Nizar-Rahme — CHANGES_REQUESTED** [review]

File: src/features/winterStorageApplicationList/utils.ts (hunk 1 above)
Comment (TatuArvela) [reply src/features/winterStorageApplicationList/utils.ts:54]: Let's use the same style for all, so if this suggestion is what the team prefers, I can change all the utils.

Comment (Nizar-Rahme) [reply src/features/winterStorageApplicationList/utils.ts:54]: Yeah, but for me, it's a matter of avoiding `Type Assertion`.

Comment (TatuArvela) [reply src/features/winterStorageApplicationList/utils.ts:54]: Changed

**Review by Nizar-Rahme — APPROVED** [review]
LGTM! 💯

**Pull request merged**